operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -1284,5 +763,516 @@ spec:
         path: conditions
         x-descriptors:
         - urn:alm:descriptor:io.kubernetes.conditions
-      version: v1alpha1
+      version: v1beta2
+  description: >
+    **Red Hat AMQ Streams** is a massively scalable, distributed, and high performance data streaming platform based on the Apache Kafka® project.
+    AMQ Streams provides an event streaming backbone that allows microservices and other application components to exchange data with extremely high throughput and low latency.
+
+    **The core capabilities include:**
+
+    * A pub/sub messaging model, similar to a traditional enterprise messaging system, in which application components publish and consume events to/from an ordered stream
+
+    * The long term, fault-tolerant storage of events
+
+    * The ability for a consumer to replay streams of events
+
+    * The ability to partition topics for horizontal scalability
+
+    ### CRD Upgrades
+
+    **!!! IMPORTANT !!!** 
+    This release introduces new API version `v1beta2` to all Strimzi custom resources. 
+    This is a preparation for migration to `apiextensions/v1` which is needed because Kubernetes 1.22 will remove support for `apiextensions/v1beta1`. 
+    Migration to `v1beta2` needs to be completed for all Strimzi CRDs and CRs after the upgrade to 0.22 is done and before upgrading to Strimzi 0.23 which will support only Strimzi `v1beta2` APIs and `apiextensions/v1` CRDs.
+    For more details about the CRD upgrades, see the [documentation](https://strimzi.io/docs/operators/0.22.1/deploying.html#assembly-upgrade-resources-str).
```

Comment:
This has to link to downstream docs (but 1.7 downstream docs - we screwed this last time). Ask Paul and Daniel what will be the URL for this.

Reply:
@laidan6000 Do you know what downstream link we should put here?

Reply:
https://access.redhat.com/documentation/en-us/red_hat_amq/2021.q2/html/deploying_and_upgrading_amq_streams_on_openshift/assembly-upgrade-str#assembly-upgrade-resources-str